client/src/components/menu-views/category.js

```diff
@@ -7,7 +7,6 @@ class Category extends Component {
     const { category } = this.props;
     return (
       <Container className="col-item">
-        <br />
```

Comment:
Why have you deleted here <br/>? Text is not in center thereafter.
![AddDish](https://user-images.githubusercontent.com/54036128/81480285-5710e800-9231-11ea-9075-3b703fe17624.jpg)

Reply:
I have returned it